flavors/liquid-mainnet/config.env

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 export SITE_TITLE='Liquid Explorer'
-export HOME_TITLE='Explorer'
+export HOME_TITLE='Liquid Explorer'
```

Comment:
Oh this is because in the Blockstream flavour config file we are overriding the SITE_TITLE and append "- blockstream.info" to it. We don't want to display this as the main title on the page though so that's why we specifically set the HOME_TITLE as well.

flavours/blockstream/config.env:
export SITE_TITLE="$SITE_TITLE - Blockstream.info"

Reply:
Ah ah, I see, let me fix that...

Reply:
Added back: https://github.com/Blockstream/esplora/commit/447f44e0a16f17ceb9e9fecc25bbfb62f2e85cfa